db/connection.js

```diff
@@ -1,12 +1,18 @@
 const { Pool } = require('pg');
 const ENV = process.env.NODE_ENV || 'development';
+const config = {}
 
 require('dotenv').config({
   path: `${__dirname}/../.env.${ENV}`,
 });
 
-if (!process.env.PGDATABASE) {
-  throw new Error('PGDATABASE not set');
+if (!process.env.PGDATABASE && !process.env.DATABASE_URL) {
+  throw new Error('PGDATABASE or DATABASE_URL not set');
+}
+
+if(ENV === 'production'){
+  config.connectionString = process.env.DATABASE_URL
+  config.max = 2
 }
 
 module.exports = new Pool();
```

Comment:
Really good refactor but dont forget to provide config as the argument for pool as well